### ISSUE TITLE
Question about vrcnet result

### ISSUE THREAD

**Issue by yolanehe** [paul007pl/VRCNet#17]
Hi, there. I am confused about the result of the vrcnet. I trained it using the parameters in the cfgs file (2048pts, 100 epoch, cd loss), but the result (cd_p: 0.174726 cd_t: 0.105811 emd: 0.277059 f1: 0.000139 ) seems quite different from what you reported in the paper. Also, the visible result is quite weird...

So, I was wondering what's the problem? Is the pretrained model necessarily needed? Or is the parameters in the cfgs file are not exactly what you guys use in the experiment?

Still, thanks for sharing the code. It is definitely incredible work.

 Looking forward to your reply~

**Comment by yolanehe** [comment]
I think I have found out the reason, thanks~
